tensorflow_gan/python/losses/losses_impl.py

```diff
@@ -1155,3 +1157,83 @@ def cycle_consistency_loss(data_x,
       tf.compat.v1.summary.scalar('cycle_consistency_loss', loss)
 
   return loss
+
+
+def ragan_discriminator_loss(d_real, 
+                             d_fake,
```

Comment:
Can you please use the same naming conventions as the other losses in this file? See any of the other discriminator loss functions for what to call these